app/src/androidTest/java/com/rentminder/ExampleInstrumentedTest.kt

```diff
@@ -19,6 +19,6 @@ class ExampleInstrumentedTest {
     fun useAppContext() {
         // Context of the app under test.
         val appContext = InstrumentationRegistry.getInstrumentation().targetContext
-        assertEquals("com.rentminder", appContext.packageName)
+        assertEquals("com.reminder", appContext.packageName)
```

Comment:
This breaks the test.  The package is indeed rentminder, not reminder, as the project is rent minder.

Leave this as it were.